config/routes.rb

```diff
@@ -42,6 +42,7 @@
   post '/mfa-enrolment', to: 'mfa#enrol', as: :enrol_to_mfa
 
   get 'profile', to: 'profile#show'
+  post 'profile/change-password', to: 'profile#change_password'
```

Comment:
as per above, there should be get method too as the password change should be on a separate page